networks/p2p/discover/ntp.go

```diff
@@ -31,8 +31,9 @@ import (
 )
 
 const (
-	ntpPool   = "pool.ntp.org" // ntpPool is the NTP server to query for the current time
-	ntpChecks = 3              // Number of measurements to do against the NTP server
+	ntpPool          = "pool.ntp.org" // ntpPool is the NTP server to query for the current time
+	ntpChecks        = 3              // Number of measurements to do against the NTP server
+	ntpChecksForBoot = 5
```

Comment:
```suggestion
	ntpChecks        = 5              // Number of measurements to do against the NTP server
```
If we seek robustness, why not just use `5` in both cases?